AirCasting/SessionViews/StreamsView.swift

```diff
@@ -36,14 +36,27 @@ struct StreamsView: View {
                 .padding(.bottom, 3)
                 .padding(.horizontal)
             if let dbStream = session.dbStream {
-                SingleMeasurementView(stream: dbStream,
-                                      value: lastMicMeasurement(),
-                                      threshold: threshold,
-                                      selectedStream: .constant(dbStream),
-                                      measurementPresentationStyle: measurementPresentationStyle)
+                if let threshold = thresholdFor(stream: dbStream) {
+                    SingleMeasurementView(stream: dbStream,
+                                          value: lastMicMeasurement(),
+                                          threshold: threshold,
+                                          selectedStream: .constant(dbStream),
+                                          measurementPresentationStyle: measurementPresentationStyle)
+                }
             }
         }
     }
+    
+    func thresholdFor(stream: MeasurementStreamEntity?) -> SensorThreshold? {
```

Comment:
I see this in multiple places, let's try to unify since the logic is basically the same everywhere. See my proposition below 😀